scripts/obtain/dotnet-install.sh

```diff
@@ -245,16 +259,24 @@ check_pre_reqs() {
     fi
 
     if [ "$(uname)" = "Linux" ]; then
-        if [ ! -x "$(command -v ldconfig)" ]; then
-            echo "ldconfig is not in PATH, trying /sbin/ldconfig."
-            LDCONFIG_COMMAND="/sbin/ldconfig"
+        if is_musl_based_distro; then
+            if ! command -v scanelf > /dev/null; then
+                say_warning "scanelf not found, please install pax-utils package."
+                return 0
+            fi
+            LDCONFIG_COMMAND="scanelf --ldpath -BF '%f'"
+            [ -z "$($LDCONFIG_COMMAND 2>/dev/null | grep libintl)" ] && say_warning "Unable to locate libintl. Probable prerequisite missing; install libintl (or gettext)."
```

Comment:
Is 'libintl' required to run the .NET Core platform on all musl variants; the majority of the variants?

Reply:
In the base system or minimal install of Alpine Linux and Void Linux, `libintl` is not installed. If we successfully install dotnet, and run the first command (e.g. `dotnet new`), we get:

```
Failed to load , error: Error loading shared library libintl.so.8: No such file or directory (needed by /root/.dotnet/shared/Microsoft.NETCore.App/3.0.0/libcoreclr.so)
Failed to bind to CoreCLR at '/root/.dotnet/shared/Microsoft.NETCore.App/3.0.0/'
Failed to create CoreCLR, HRESULT: 0x80008088
```

With this addition, user gets install-time error about the missing dependency (like this script already has for libunwind etc.). If some other musl-libc based distro or full version of Alpine/Void already has this lib installed, this condition will be skipped.